packages/web-react/src/components/Link/README.md

```diff
@@ -8,12 +8,13 @@ Link allows users to follow navigation.
 
 ## Available props
 
-| Name           | Type                                             | Description                                     |
-| -------------- | ------------------------------------------------ | ----------------------------------------------- |
-| `href`         | `string`                                         | Link's href attribute                           |
-| `color`        | [Action Link Color dictionary][dictionary-color] | Color of the link                               |
-| `isUnderlined` | `boolean`                                        | Whether is the link underlined, default `false` |
-| `isDisabled`   | `boolean`                                        | Wheter is the link disabled, default `false`    |
+| Name           | Type                                             | Default   | Required | Description                                     |
+| -------------- | ------------------------------------------------ | --------- | -------- | ----------------------------------------------- |
+| `href`         | `string`                                         | -         | ✕        | Link's href attribute                           |
+| `color`        | [Action Link Color dictionary][dictionary-color] | `primary` | ✕        | Color of the link                               |
+| `isUnderlined` | `boolean`                                        | `false`   | ✕        | Whether is the link underlined, default `false` |
+| `isDisabled`   | `boolean`                                        | `false`   | ✕        | Whether is the link disabled, default `false`   |
```

Comment:
```suggestion
| `isDisabled`   | `boolean`                                        | `false`   | ✕        | Whether is the link disabled   |
```
it is already said in the previous column

Reply:
done